e2e/scripts/common.sh

```diff
@@ -0,0 +1,21 @@
+wait_for_healthy_node() {
```

Comment:
thank you for finally making a common script file
there's a lot of stuff that should get moved here